cmd/kured/main.go

```diff
@@ -96,6 +99,8 @@ func main() {
 		"slack username for reboot notfications")
 	rootCmd.PersistentFlags().StringVar(&slackChannel, "slack-channel", "",
 		"slack channel for reboot notfications")
+	rootCmd.PersistentFlags().StringVar(&notifyURL, "notify-url", "",
+		"notify URL for reboot notfications")
```

Comment:
I guess we will need to expose this in the helm chart as well, right?

Reply:
I think the normal pattern is to update the helm chart with a separate PR after the functional change has merged.

Reply:
wfm!